src/lib/validators/openApiDiff.ts

```diff
@@ -250,16 +226,24 @@ export class OpenApiDiff {
     const outputFolder = await fs.promises.mkdtemp(path.join(os.tmpdir(), "oad-"))
     const outputFilePath = path.join(outputFolder, `${outputFileName}.json`)
     const outputMapFilePath = path.join(outputFolder, `${outputFileName}.map`)
-    // Cross-platform shell argument escaping - behavior is validated in shellEscapingTest.ts
-    const autoRestCmd = tagName
-      ? `${this.autoRestPath()} ${escapeShellArg(swaggerPath)} --v2 --tag=${escapeShellArg(tagName)} --output-artifact=swagger-document.json` +
-        ` --output-artifact=swagger-document.map --output-file=${escapeShellArg(outputFileName)} --output-folder=${escapeShellArg(outputFolder)}`
-      : `${this.autoRestPath()} --v2 --input-file=${escapeShellArg(swaggerPath)} --output-artifact=swagger-document.json` +
-        ` --output-artifact=swagger-document.map --output-file=${escapeShellArg(outputFileName)} --output-folder=${escapeShellArg(outputFolder)}`
 
-    log.debug(`Executing: "${autoRestCmd}"`)
+    const { file: autoRestFile, args: autoRestArgs } = this.autoRestFileArgs()
+
+    const swaggerArgs = tagName ? [swaggerPath, `--tag=${tagName}`] : [`--input-file=${swaggerPath}`]
```

Comment:
Do we still need to quote/escape any of the variable strings used as values in params like `--foo=${bar}`?  Or would a param like `--foo=C:\folder with spaces\` work without quoting, since it's not going through the shell?